crates/core/test_migrations/2022-08-04-200854_seed_nfts/down.sql

```diff
@@ -0,0 +1,27 @@
+delete from metadatas where address = any(array [
```

Comment:
Normally unit tests involving databases are done in transaction that leave the db empty after each unit test but since we are only doing read operations seeding with a shared set of data works for our use case.

Reply:
Seconding this. I like the migration setup. It would be awesome to be able to define a migration per test in such a way that we could run `up` and `down` for a specific migration for each test, just to keep the db minimally populated with test-specific data.

This is a great start, though!